.gitlab-ci.yml

```diff
@@ -720,6 +722,22 @@ library:ci-fiat_crypto:
   - library:ci-coqprime
   - plugin:ci-rewriter
 
+# We cannot use flambda due to
+# https://github.com/ocaml/ocaml/issues/7842, see
+# https://github.com/coq/coq/pull/11916#issuecomment-609977375
+library:ci-fiat_crypto_ocaml:
+  extends: .ci-template
+  stage: stage-5
+  needs:
+  - build:edge+flambda
+  - library:ci-coqprime
+  - library:ci-fiat_crypto
+  - plugin:ci-bignums
+  - plugin:ci-rewriter
+  dependencies:
+  - build:edge+flambda
+  - library:ci-fiat_crypto
```

Comment:
Does this job actually depend on all this stuff or just fiat_crypto?

Reply:
I need the fiat-crypto build artifact and the OCaml compiler.  I thought nneds was supposed to include transitive dependencies?  Or are you asking about the dependency on build?

Reply:
needs s supposed to include transitive dependencies because coq dependencies are transitive. IIUC this job doesn't use coqc so the transitive dependencies shouldn't be needed.

Reply:
Ah, neat.  Now updated.  I've left build+edge in there because I think I need coq_makefile, though I could be wrong.